### DISCUSSION TITLE
Action StretchSplitTo{Right/Left/Top/Bottom} act only once using script

### DISCUSSION THREAD

**Discussion by Marchonq** [JetBrains/ideavim#653] [Q&A]
I want to execute the StretchSplitToLeft action for multiple times, but seems my ide only acted once instead of multiple times.
It is ok if I change the action to others, dose anyone have samilar problems?
I tested my Rider and Pycharm, both fialed.
```
function! StretchRight(total)
    let l:c = 0
    while l:c < a:total
      execute ':action StretchSplitToLeft'
      let l:c += 1
    endwhile
endfunction
```

**Comment by citizenmatt** [comment]
Unfortunately, there's no workaround for this, as actions are executed asynchronously.
